style/main.css

```diff
@@ -194,6 +194,30 @@ form[role="dialog"][data-type="confirm"].lcdui-alert section {
   left: calc(50% - 17px);
 }
 
+.background-screen {
+  z-index: 2;
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: auto;
+  bottom: auto;
+  background-color: white;
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-image: url("../img/icon-128.png");
```

Comment:
Maybe for some reason the image wasn't in the package of the app you pushed to the device?